cumulus_etl/common.py

```diff
@@ -32,49 +31,13 @@ def ls_resources(root, resource: str) -> list[str]:
 #
 ###############################################################################
 
-_user_fs_options = {}  # don't access this directly, use get_fs_options()
```

Comment:
This bit of business with tracking use-filesystem options (like --s3-region) got moved to `store` because that flips the relationship between them - now `common` uses `store` and can make use of `store.Root`. (and it makes more sense for these FS options to live there)